ci/citest/acceptance-test/multibox/10.0.100.15-vdc-executor-lxc-ovs/guestroot/etc/sysconfig/network-scripts/ifcfg-ovs0

```diff
@@ -1,4 +1,4 @@
-DEVICE=br0
+DEVICE=ovs0
```

Comment:
If you want to change the bridge name, don't forget to update `ifcfg-eth1` and the draw.io illustrations as well.